applications/stepswitch/src/stepswitch_inbound.erl

```diff
@@ -282,6 +289,26 @@ transition_port_in(Number, JObj) ->
 %%
 %% @end
 %%--------------------------------------------------------------------
+
+%% string: two or more zeros -> true
+-spec is_zero(binary() | undefined) -> boolean().
```

Comment:
Note: this should be equivalent to `api_ne_binary()` (not super important, but for the future :) )